<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Inventory</class>
 <widget class="QWidget" name="Inventory">
  <property name="windowModality">
   <enum>Qt::WindowModality::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1160</width>
    <height>759</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Food Items</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	color: black;
	background-color: rgb(237, 233, 230);
}</string>
  </property>
  <layout class="QGridLayout" name="mainLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
     </property>
     <property name="topMargin">
      <number>36</number>
     </property>
     <item>
      <widget class="QWidget" name="widget_2" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout_13">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QWidget" name="productsCatalog_widget" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>10</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>10</width>
            <height>80</height>
           </size>
          </property>
          <property name="sizeIncrement">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(0, 153, 73);</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="productsCatalog_button">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Segoe UI Variable</family>
            <pointsize>15</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
  background-color: transparent;
  border: none;
 color: rgb(0, 0, 0);
  text-align: center;
  padding-left: 20px;
  padding-right: 20px;
}

QPushButton:hover {
  background-color: transparent;     /* Fondo al pasar el mouse */
  color: rgb(255, 255, 255);             /* Color del texto al pasar el mouse */
}

QPushButton:pressed {
  background-color: transparent;     /* Fondo al presionar */
  color: rgb(0, 153, 73);
          /* Color del texto al presionar */
}

QPushButton:disabled {
  background-color: transparent;     /* Fondo deshabilitado */
  color: #7f8c8d;                /* Color del texto deshabilitado */
}

QPushButton:checked {
  background-color: transparent;     /* Fondo cuando está checkable y activado */
  color: rgb(0, 153, 73);
}
</string>
          </property>
          <property name="text">
           <string>Productos</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_14">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="categoriesCatalog_widget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>10</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>10</width>
           <height>80</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="categoriesCatalog_button">
         <property name="font">
          <font>
           <family>Segoe UI Variable</family>
           <pointsize>15</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LayoutDirection::LeftToRight</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
  background-color: transparent;
  border: none;
 color: rgb(0, 0, 0);
  text-align: center;
  padding-left: 20px;
  padding-right: 20px;
}

QPushButton:hover {
  background-color: transparent;     /* Fondo al pasar el mouse */
  color: rgb(255, 255, 255);             /* Color del texto al pasar el mouse */
}

QPushButton:pressed {
  background-color: transparent;     /* Fondo al presionar */
  color: rgb(0, 153, 73);
          /* Color del texto al presionar */
}

QPushButton:disabled {
  background-color: transparent;     /* Fondo deshabilitado */
  color: #7f8c8d;                /* Color del texto deshabilitado */
}

QPushButton:checked {
  background-color: transparent;     /* Fondo cuando está checkable y activado */
  color: rgb(0, 153, 73);
}
</string>
         </property>
         <property name="text">
          <string>Categorías</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_15">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="suppliesCatalog_widget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>10</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>10</width>
           <height>40</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="suppliesCatalog_button">
         <property name="font">
          <font>
           <family>Segoe UI Variable</family>
           <pointsize>15</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
  background-color: transparent;
  border: none;
 color: rgb(0, 0, 0);
  text-align: center;
  padding-left: 20px;
  padding-right: 20px;
}

QPushButton:hover {
  background-color: transparent;     /* Fondo al pasar el mouse */
  color: rgb(255, 255, 255);             /* Color del texto al pasar el mouse */
}

QPushButton:pressed {
  background-color: transparent;     /* Fondo al presionar */
  color: rgb(0, 153, 73);
          /* Color del texto al presionar */
}

QPushButton:disabled {
  background-color: transparent;     /* Fondo deshabilitado */
  color: #7f8c8d;                /* Color del texto deshabilitado */
}

QPushButton:checked {
  background-color: transparent;     /* Fondo cuando está checkable y activado */
  color: rgb(0, 153, 73);
}
</string>
         </property>
         <property name="text">
          <string>Suministros</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
